src/main/java/com/raisetech/restapi/NameController.java

```diff
@@ -12,27 +14,31 @@
 @RestController
 public class NameController {
     @GetMapping("/names")
-    public List<String> getNames() {
+    public List<String> getNames(String name) {
         return List.of("koyama", "tanaka");
     }
 
     @PostMapping("/names")
-    public ResponseEntity<String> create(@RequestBody CreateForm form) {
+    public ResponseEntity<String> create(@Validated @RequestBody CreateForm form, BindingResult bindingResult) {
```

Comment:
下記のようにすればBad Requestのときに自動でエラーを作成してレスポンスとして返してくれるはず。
```suggestion
    public ResponseEntity<String> create(@Validated @RequestBody CreateForm form) {
```